snapshot/win/process_reader_win.cc

```diff
@@ -125,25 +126,102 @@ HANDLE OpenThread(
   return handle;
 }
 
+void DoStackWalk(ProcessReaderWin::Thread* thread,
+                 HANDLE process,
+                 HANDLE thread_handle,
+                 bool is_64_reading_32) {
+  if (is_64_reading_32) {
+    // TODO: we dont support it right away, maybe in the future
+    return;
+  }
```

Comment:
if i understand correctly, this flag only affects the logic in `ARCH_CPU_X86_64` and possible `ARCH_CPU_ARM64`, but is ignored otherwise for other architectures. 

do we want to be short-circuiting here if for some reason we're on `ARCH_CPU_X86` yet `is_64_reading_32` is set to true?

Reply:
I don’t think that can ever be the case. That boolean flag I think is specifically for the case when a 64bit handler is inspecting a 32bit child.